lib/job.js

```diff
@@ -32,6 +32,24 @@ async function findMetrics(build, stepName, aggregateInterval) {
     return metrics;
 }
 
+/**
+ * @description Gets the token from token gen fn
+ * @param {Object} self -> this object
+ * @param {String} jobId
+ * @param {Object} pipeline
+ * @return {String} A jwt token
+ */
+function getToken(self, pipeline, jobId) {
```

Comment:
move this to https://github.com/screwdriver-cd/models/blob/master/lib/helper.js ?
this seems to be duplicated in jobFactory as well